src/sentry/models/files/utils.py

```diff
@@ -29,8 +34,7 @@ class nooplogger:
     exception = staticmethod(lambda *a, **kw: None)
 
 
-def get_size_and_checksum(fileobj, logger=nooplogger):
```

Comment:
the logger argument was never used -- also this is set up really weirdly so I'm going to try and kill `nooplogger` separately.  kinda the point of the logging system is they're noops by default without needing to do this